src/App.vue

```diff
@@ -146,7 +158,9 @@ export default {
 
       // Add slight delay so the progress bar makes
       // it to 100% before disappearing
-      setTimeout(() => (this.loading = false), 300);
+      await delay(300);
+      this.loading = false;
+      this.bitcoinStarted = false;
```

Comment:
Why do you think it's required to reset it?

Reply:
Tbh I wasn't sure. I didn't initially then decided to add it.

Will the loading view only ever be displayed when the dashboard is reloaded? Or can something else cause it be shown after it's been shown initially like if Bitcoin/LND goes down?

Reply:
Ahh yes, that makes sense. The loading view does take over if any of the containers goes down.

Shall we reset `bitcoinStarted` right after the conditional block `if (this.loadingProgress <= 60)` ends? Since we also do not have to wait for `300ms` to reset it as the delay was only added for the progress bar to make it to 100% before disappearing.

Reply:
Sure, will do.